tests/channels/test_d0_to_kskpkm.py

```diff
@@ -12,7 +12,7 @@ def test_script():
     logging.getLogger().setLevel(logging.INFO)
     # initialize the graph edges (initial and final state)
     initial_state = [("D0", [0])]
-    final_state = [("K_S0", [0]), ("K+", [0]), ("K-", [0])]
+    final_state = [("K0bar", [0]), ("K+", [0]), ("K-", [0])]
```

Comment:
why you took `K0bar` and not `K0`?

Reply:
Eamm, random choice

Reply:
I see, `K0` would have been more straight forward for me. But ok I wont be that picky